.github/workflows/discord-i18n-ping.yml

```
@@ -0,0 +1,35 @@
+name: Translation Tuesday
+
+on:
+  schedule:
+    - cron:  '0 0 * * TUE'
+  workflow_dispatch:
+
+jobs:
+  translation:
+    name: translation_tuesday
+    if: ${{ github.repository_owner == 'withastro' && github.event.commits[0].message != '[ci] format' }}
```

```suggestion
    if: ${{ github.repository_owner == 'withastro' }}
```
As the script is not being run on a commit, but by a cron job or manually, we shouldn't try to reference a commit here.